### PR TITLE
Add OMIA species specific terms

### PR DESCRIPTION
**Pull request by katiermullen**
addresses #7225

### CODE REVIEW

**Review by sabrinatoro — APPROVED** [review]
@katiermullen There are a few things to be updated.
I am approving, so you are not blocked, but please update where I added comments. 
Note: If I am getting there before you, I will do it (whoever gets there first) :-)

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -541546,12 +541546,16 @@ id: MONDO:0700161
...
 is_a: MONDO:0700139 {source="NCIT:C158783"} ! canine neoplasm
+is_a: MONDO:1011440 {source="OMIA:001516", source="https://orcid.org/0000-0002-5002-8648"} ! gastrointestinal stromal tumour, non-human animal
 intersection_of: MONDO:0005583 ! non-human animal disease
 intersection_of: in_taxon NCBITaxon:9615 ! Canis lupus familiaris
 intersection_of: MONDO:0700097 MONDO:0011719 ! cross-species analog gastrointestinal stromal tumor
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
-property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/5252" xsd:anyURI
+relationship: in_taxon NCBITaxon:9615 {source="OMIA:001516-9615", source="https://orcid.org/0000-0002-5002-8648"} ! Canis lupus familiaris
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/5252" xsd:string
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:string
```
[the remark this answers is not in the document]
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:541558]: done

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -568028,8 +568035,2021 @@ def: "Ligneous membranitis that occurs in non-human animals." [MONDO:patterns/no
...
 is_a: MONDO:0005583 {source="OMIA:002020"} ! non-human animal disease
 intersection_of: MONDO:0005583 ! non-human animal disease
-intersection_of: MONDO:0700097 MONDO:0005338 ! cross-species analog open-angle glaucoma
+intersection_of: MONDO:0700097 MONDO:0100560 ! cross-species analog ligneous conjunctivitis
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011460
+name: alzheimer disease, degu
```
[the remark this answers is not in the document]
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:568044]: done

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -568028,8 +568035,2021 @@ def: "Ligneous membranitis that occurs in non-human animals." [MONDO:patterns/no
...
+is_a: MONDO:1011443 {source="OMIA:000033", source="https://orcid.org/0000-0002-5002-8648"} ! Alzheimer disease, non-human animal
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: in_taxon NCBITaxon:10160 ! Octodon degus
+intersection_of: MONDO:0700097 MONDO:0004975 ! cross-species analog Alzheimer disease
+relationship: in_taxon NCBITaxon:10160 {source="OMIA:000033-10160", source="https://orcid.org/0000-0002-5002-8648"} ! Octodon degus
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011461
+name: alzheimer disease, dog
```
[the remark this answers is not in the document]
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:568056]: done

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -568028,8 +568035,2021 @@ def: "Ligneous membranitis that occurs in non-human animals." [MONDO:patterns/no
...
+is_a: MONDO:1011443 {source="OMIA:000033", source="https://orcid.org/0000-0002-5002-8648"} ! Alzheimer disease, non-human animal
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: in_taxon NCBITaxon:9615 ! Canis lupus familiaris
+intersection_of: MONDO:0700097 MONDO:0004975 ! cross-species analog Alzheimer disease
+relationship: in_taxon NCBITaxon:9615 {source="OMIA:000033-9615", source="https://orcid.org/0000-0002-5002-8648"} ! Canis lupus familiaris
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011462
+name: alzheimer disease, domestic cat
```
[the remark this answers is not in the document]
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:568068]: done

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -568028,8 +568035,2021 @@ def: "Ligneous membranitis that occurs in non-human animals." [MONDO:patterns/no
...
+is_a: MONDO:1011443 {source="OMIA:000033", source="https://orcid.org/0000-0002-5002-8648"} ! Alzheimer disease, non-human animal
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: in_taxon NCBITaxon:9685 ! Felis catus
+intersection_of: MONDO:0700097 MONDO:0004975 ! cross-species analog Alzheimer disease
+relationship: in_taxon NCBITaxon:9685 {source="OMIA:000033-9685", source="https://orcid.org/0000-0002-5002-8648"} ! Felis catus
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011463
+name: alzheimer disease, sheep
```
[the remark this answers is not in the document]
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:568080]: done

**Comment by katiermullen** [comment]
@sabrinatoro thank you for updating the capitalization and GH tracker links!

**Pull request merged**